### ISSUE TITLE
Compiler Error C2220  warning treated as error - no object file generated

### ISSUE THREAD

**Issue by xbelanch** [kolumb/zoomer#1]
Trying to build it I get this error:

````
R:\zoomer>enable_msvc.bat
**********************************************************************
** Visual Studio 2017 Developer Command Prompt v15.9.33
** Copyright (c) 2017 Microsoft Corporation
**********************************************************************
[vcvarsall.bat] Environment initialized for: 'x64'

R:\zoomer>build_msvc.bat
cl : Línea de comandos warning D9002 : se omite la opción desconocida '/std:c11'
zoomer.c
r:\zoomer\src\zoomer_vector.h(13): error C2220: advertencia tratada como error; ningún archivo 'object' generado
r:\zoomer\src\zoomer_vector.h(13): warning C4204: se ha utilizado una extensión no estándar: inicializador de agregado no constante
r:\zoomer\src\zoomer_vector.h(25): warning C4204: se ha utilizado una extensión no estándar: inicializador de agregado no constante
r:\zoomer\src\zoomer_vector.h(37): warning C4204: se ha utilizado una extensión no estándar: inicializador de agregado no constante
r:\zoomer\src\zoomer_vector.h(43): warning C4204: se ha utilizado una extensión no estándar: inicializador de agregado no constante
r:\zoomer\src\zoomer_vector.h(49): warning C4204: se ha utilizado una extensión no estándar: inicializador de agregado no constante
r:\zoomer\src\zoomer_navigation.c(4): warning C4204: se ha utilizado una extensión no estándar: inicializador de agregado no constante
r:\zoomer\src\zoomer_main.c(379): warning C4204: se ha utilizado una extensión no estándar: inicializador de agregado no constante
r:\zoomer\src\zoomer_main.c(441): warning C4204: se ha utilizado una extensión no estándar: inicializador de agregado no constante
r:\zoomer\src\zoomer_main.c(442): warning C4204: se ha utilizado una extensión no estándar: inicializador de agregado no constante
`````

**Comment by xbelanch** [comment]
Following the recommended techniques (https://docs.microsoft.com/en-us/cpp/error-messages/compiler-errors-1/compiler-error-c2220?view=msvc-160), I changed /W4 for /W3.  Trying to compile it again it raises a new error:

```
R:\zoomer>build_msvc.bat
cl : Línea de comandos warning D9002 : se omite la opción desconocida '/std:c11'
zoomer.c
R:\zoomer\src\zoomer.c : fatal error C1083: No se puede abrir el archivo generado por el compilador: 'R:\zoomer\build\zoomer.obj': No such file or directory
```

I created manually the missing `build` directory and now it compiles correctly.

**Comment by kolumb** [comment]
Thank you for reporting this issue!
I tried to fix those warnings in latest commit cec65a3d0b92ee5d586b896c9448914b1e8f2536.
I was not fixing `Línea de comandos warning D9002 : se omite la opción desconocida '/std:c11'` but it shouldn't break build in this version of compiler.
